application/models/QuestionTheme.php

```diff
@@ -82,6 +82,17 @@ public function relations()
         return array();
     }
 
+    /** @inheritdoc */
+    public function scopes()
+    {
+        return array(
+            'core' => array(
```

Comment:
Woulnd't be good to have a scope for user ones?

Reply:
Rename to "base"?

Reply:
Why rename? Is there a benefit?